app/server/api/search_api.py

```diff
@@ -85,22 +163,47 @@ def get(self):
 
             # Then use those results to join aginst TransferAccount or CreditTransfer
             if search_type == 'transfer_accounts':
+            # TransferAccount Search Logic
                 final_query = db.session.query(TransferAccount)\
                     .join(user_search_result, user_search_result.c.default_transfer_account_id == TransferAccount.id)\
-                    .order_by(db.text('rank DESC'))\
+                    .join(User, user_search_result.c.default_transfer_account_id == User.default_transfer_account_id)\
+                    .execution_options(show_all=True)\
                     .filter(user_search_result.c.rank > 0.0)\
                     .filter(TransferAccount.is_ghost != True)
+                if sort_by_arg == 'rank':
+                    final_query = final_query.order_by(order(user_search_result.c.rank))
+                else:
+                    final_query = final_query.order_by(order(sort_by))
```

Comment:
Nice, didn't realize you could do that `final_query.order_by(order(sort_by))`... I was like where is order defined !?